x-pack/plugins/fleet/public/plugin.ts

```diff
@@ -76,6 +80,7 @@ export interface FleetSetupDeps {
 export interface FleetStartDeps {
   data: DataPublicPluginStart;
   navigation: NavigationPublicPluginStart;
+  customIntegrations: CustomIntegrationsStart;
```

Comment:
Is this dependency required only for Storybook?

Reply:
No, it's used in x-pack/plugins/fleet/public/applications/integrations/app.tsx